lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js

```diff
@@ -163,89 +164,109 @@ test('When all previous required questions have been answered - and a url is ret
   t.end()
 })
 
-test.skip('it attaches a user submission when the property is set to true', async t => {
-  getUserDataPropertyStub.resetHistory()
-  getUserDataPropertyStub.withArgs('email').returns('user@example.com')
+test('it attaches a user submission when the property is set to true', async t => {
   getInstancePropertyStub.withArgs('service', 'attachUserSubmission').returns(true)
 
   const pageSummaryController = proxyquire('./page.summary.controller', {
     '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
     '../../../../presenter/pdf-payload': {pdfPayload: pdfPayloadStub},
-    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub}
+    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub},
+    '../../../../constants/constants': {
+      SERVICE_OUTPUT_EMAIL: 'bob@gov.uk'
```

Comment:
Is setting this constant required for this assertion?

Reply:
Yeah it is strangely